requirements.yml

```diff
@@ -4,3 +4,4 @@
 
 - src: defunctzombie.coreos-bootstrap
   name: coreos_bootstrap
+  when: onpremise|bool
```

Comment:
will this work in this context? since this is for doing a ```ansible-galaxy install``` . Do we not need to set the variable in the playbook (site.yml) ?